src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -389,6 +389,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
 #End Region
 
 #Region "Buffer helpers"
+
```

Comment:
Have you measured performance effects of this change? Is this method being inlined by JIT?